lib/Importer/ONNXModelLoader.cpp

```diff
@@ -605,7 +605,7 @@ llvm::Error ONNXModelLoader::loadConv(const ONNX_NAMESPACE::NodeProto &op,
   if (op.input_size() > 2) {
     auto &biasTensorName = op.input(2);
     // Load the serialized bias vector.
-    bias = getConstantByNameOrNull(biasTensorName);
+    ASSIGN_VALUE_OR_RETURN_ERR(bias, getConstantByName(biasTensorName));
```

Comment:
i do not think this fixes a problem.

getConstantByName does not return error in any case.


Also, we' need to have a unit test in ONNXModelLoader to make sure we cover the erroneous code path.

Reply:
I actually overlooked `getConstantByName` and thought it's getConstantByNameOrNull.
This seems to be fine.

Reply:
Thanks! I added a test to make sure it does. The test passes as-is but fails when I change back to getConstantByNameOrNull.